modules/agentless-s3-bucket/variables.tf

```diff
@@ -0,0 +1,26 @@
+
+variable "tags" {
+  description = "A map of additional tags to add to the IAM role/profile created"
+  type        = map(string)
+  default     = {}
+}
+
+variable "iam_delegate_role_arn" {
+  description = "Reference ARN to the Agentless delegate role"
```

Comment:
This reads weird, how about `ARN of the Agentless RDS IAM role`? (or adjusted if needed)